src/phoenix/server/api/helpers/playground_clients.py

```diff
@@ -707,7 +707,6 @@ async def chat_completion_create(
             "messages": anthropic_messages,
             "model": self.model_name,
             "system": system_prompt,
-            "max_tokens": 1024,
```

Comment:
It seems like we throw either way when it snot in input (not sure if there is something in my environment). So we could remove this, or return it as the default. It's only our default though not theres so not sure how we feel about that